feed-integration-webapp/src/main/resources/atomfeed.properties

```diff
@@ -11,6 +11,6 @@ feed.failedEventMaxRetry=10
 openmrs.auth.uri=http://localhost:8080/openmrs/ws/rest/v1/session
```

Comment:
are this properties externalizable? application.properties can be, as spring takes the default name for config location .. but these properties can not be hardcoded

Reply:
So actually, there is a separate branch, BAH-1938, where crater-atomfeed has been dockerised. In that branch, there are property file templates which replace the property files, thus its externalised. So for the main branch, we have two options: we can keep the keys in each property file which no value and they get replaced by the template property files or we can delete the property files completely and let docker take care of it while creating the container using the templates. Please let me know which one should be done. 
BAH-1938 also has older source code but dockerfile should be upto date.

Reply:
I will let @N0-man comment on this. 

My thoughts:
1. We have to merge BAH-1938 into main. Can't have code across branches. 
2. Main branch should have properties files with some defaults, but with the ability to override (templatize).
3. No passwords.
4. Definitely should work in docker-compose / minikube. Need to support on-prem and cloud deployments.

Reply:
1. I am working on that now.
2. Templates will override the whole file as is. I will keep some defaults and the ability to override it using templates.
3. Passwords will be removed
4. Project has not been configured for minikube. Will support on-perm and cloud deployments.

Reply:
Created a PR for merging docker changes 

https://github.com/Bahmni/crater-atomfeed/pull/4